packages/calcite-components/src/components/action-bar/action-bar.scss

```diff
@@ -3,91 +3,54 @@
  *
  * These properties can be overridden using the component's tag as selector.
  *
- * @prop --calcite-action-bar-action-background-color: defines the background color of an action sub-component inside the component.
```

Comment:
I may be missing something, but it seems like it is no longer possible to theme some of the internal components:

* action pad/bar ➡️ action-group
* action menu ➡️ action, popover
* action group ➡️ action, action-menu

Reply:
oop. thanks for the catch. adding those back